pkg/publish/servicelb.go

```diff
@@ -33,6 +35,21 @@ func ServiceLoadBalancer(svc *v1.ServiceInstance) (result []kclient.Object, _ er
 		return nil, nil
 	}
 
+	cfg, err := config.Get(req.Ctx, req.Client)
+	if err != nil {
+		return nil, err
+	}
+
+	if cfg.ServiceLBAnnotations != nil && len(cfg.ServiceLBAnnotations) > 0 {
```

Comment:
This check is not necessary. If there is nothing to range over, then nothing will happen.

Reply:
Makes sense, good catch.